### PR TITLE
Mahashree.rajendran/getting started synthetics

### PR DESCRIPTION
**Pull request by rmshree**
### What does this PR do?
Small changes to the getting started synthetics docs

### Preview

- https://docs-staging.datadoghq.com/mahashree.rajendran/getting_started_synthetics/getting_started/synthetics/browser_test

- https://docs-staging.datadoghq.com/mahashree.rajendran/getting_started_synthetics/getting_started/synthetics/api_test

<!-- This only works if you are part of the Datadog organization and working off of a branch - it will not work with a fork.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/synthetics/api_test.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ further_reading:
 
 ## Create an API test
 
-[API tests][1] help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as response time, HTTP status code, and header or body contents. Use the [Datadog API][2] to see the full list.
+[API tests][1] help you **monitor your API endpoints and alert** you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**. Use the [Datadog API][2] to see the full list.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/api_test.md:22]:
```suggestion
[API tests][1] help you **monitor your API endpoints and alert** you when they are failing or become too slow. These tests verify that your applications are responding to requests and meet any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**.
```
And probably remove link at the end!

File: content/en/getting_started/synthetics/api_test.md [hunk 2]
```diff
@@ -32,8 +32,8 @@ In this example, an API test is created to ensure your website is constantly up
 
     - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, which is a test web application.
     - Select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
-    - Add some tags to help organize and filter tests.
-    - Select locations for testing.
+    - You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/api_test.md:35]:
```suggestion
    - You can set tags such as `env:prod` and `app:shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
```

File: content/en/getting_started/synthetics/api_test.md [hunk 3]
```diff
@@ -32,8 +32,8 @@ In this example, an API test is created to ensure your website is constantly up
...
+    - Choose from the **managed locations** to run your test from.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/api_test.md:36]:
```suggestion
    - Choose from the **Managed Locations** to run your test from.
```

File: content/en/getting_started/synthetics/api_test.md [hunk 4]
```diff
@@ -32,8 +32,8 @@ In this example, an API test is created to ensure your website is constantly up
...
     - Click the **Test URL** button.
 
 {{< img src="getting_started/synthetics/api-test-config.png" alt="API test configuration"  style="width:60%;">}}
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/api_test.md:39]: Ideally we would change that screenshot to reflect the env tag that is now first class citizen on the creation form and the 15 locations we now have. And I think we could set the width to 100% for all screenshots/videos.

File: content/en/getting_started/synthetics/browser_test.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ further_reading:
 
 ## Create a browser test
 
-[Browser tests][1] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+[Browser tests][1] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks **verify both that your applications are up and responding to requests**, and that any conditions defined in your scenarios are met.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:18]:
```suggestion
[Browser tests][1] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These tests **verify both that your applications are up and responding to requests**, and that key business transactions can be performed by users as expected.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ further_reading:
...
 
 In this example, a browser test is configured to map a user's journey from adding an item to cart to successful checkout. If any step during the browser test fails, it throws an error that is recorded in Datadog as a **Test Result**.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:20]:
```suggestion
In this example, a browser test is configured to map a user's journey from adding an item to cart to successful checkout. Each test execution is recorded in Datadog as a **Test Result**.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 7]
```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
...
 
-    - Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use `https://www.shopist.io` as a test web application.
-    - Name the test.
-    - You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
-    - Choose devices and locations for testing. In this example, the test is only run on **Large Laptops** and on **English speaking countries**.
-    - Specify a test frequency.
-    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
+    - **Starting URL**: Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use `https://www.shopist.io` as a test web application.
+    - **Name**: Name the test.
+    - **Tags**: You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:33]:
```suggestion
    - **Tags**: You can set tags such as `env:prod` and `app:shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 8]
```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
...
+    - **Browsers & Devices**: Choose the devices you want to use for testing. In this example, the test is only run on **Large Laptops**.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:34]:
```suggestion
    - **Browsers & Devices**: Choose the devices and browsers you want to use for testing. In this example, the test is only run from **Chrome** and on **Large Laptops**.
```

File: content/en/getting_started/synthetics/browser_test.md [hunk 9]
```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
...
+    - **Locations**: Choose one of the **managed locations** to run your test from. In this example, the test is run in **Americas and Europe**.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:35]:
```suggestion
    - **Locations**: Choose one of the **Managed Locations** to run your test from. In this example, the test is run in **Americas and Europe**.
```
Let's change the below gif accordingly as well then.

File: content/en/getting_started/synthetics/browser_test.md [hunk 10]
```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
...
+    - **Specify a test frequency**: Select how often you would like the test to run.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:36]: What about a quick mention to CI here?

File: content/en/getting_started/synthetics/browser_test.md [hunk 11]
```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
...
+    - **Alert Conditions**: Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
```
Comment (MargotLepizzera) [inline content/en/getting_started/synthetics/browser_test.md:37]: Let's put fast retries first (we'll do that in the UI soon) and remove the reference about fast retries interval as well. We could even only focus on one of these options in each getting started (fast retries in browser tests and alerting rule in API test to make that section more readable).

**Comment by MargotLepizzera** [comment]
@rmshree Some more feedback in addition to the inline one:

### API tests

#### Configure

- We could update the screenshot for assertions as it's showing old components.
- Change `Now that the test is completed, if you did not create any assertions, they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.` to `After clicking Test URL, basic assertions based on your endpoint's response are automatically populated. Assertions define the alert condition and can be customized. 
In this example, three default assertions are populated when testing the URL:`
-  Change text to `To add a custom assertion, directly click on the response preview or click the New Assertion button to add an assertion manually (e.g., body contains Shop.ist).`
- Just realized we don't even mention frequency here - let's add it and mention CI integration.
- Let's put fast retries before alerting rule, and remove the reference to fast retries interval (that's an advanced option).

#### Test results

- Change `An API test homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results.` to `The API test details page includes details about the test configuration, uptime associated with the tested endpoint, historical graphs for response time and network timings, as well as the list of individual test results and events.`.
- Change `To troubleshoot a failed test, scroll to the Test Results section and click on the Test Results tab. Click on the failed test, labeled as Alert, to view detailed test results. Review the failed assertions and response details such as returned status code, response time, and associated headers and body to resolve the issue.` to `To troubleshoot a failed test, scroll to the Test Results section and click on one of the failing Test Results. Review the failed assertions and response details such as returned status code, response time, and associated headers and body to diagnose the issue.

### Browser tests

#### Configure

- Same here: ideally we could change the screenshot to an updated version with core web vitals and a 100%
### Test results

- `Manually add the Assertion`
- Step 3, breakline before note
- Change `A browser test homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results. Test results include screenshots, errors, resources, and traces for each step.` to `The browser test details page includes details about your test configuration, test uptime, historical graphs for response time and time to interactive for the first page, sample successful and failed results, and a list of test results corresponding to the selected timeframe. Each individual test result includes screenshots, core web vitals, potential errors, resources, and traces for each step.`
- Remove `The failed test step in is highlighted with a red x. Click the failed step to begin troubleshooting.`
- Add `You can start your troubleshooting by looking at the screenshots to try to understand what went wrong. Don't forget to look at screenshots of steps that happened before the failed step as these often contain the root cause of the failure.`
- The last two paragraphs could use a bit of work as well 

#### Private locations

As discussed on Friday, we could consider modifying the reference to vagrant

**Comment by apigirl** [comment]
Hello, I am closing this because it has become very outdated. Please feel free to reopen if you would like to continue working on it!

**Pull request closed without merging**